pkg/archive/archive.go

```diff
@@ -850,13 +850,19 @@ func extractTarFileEntry(path, extractDir string, hdr *tar.Header, reader io.Rea
 }
 
 // Tar creates an archive from the directory at `path`, and returns it as a
-// stream of bytes.
+// stream of bytes. This is a convenience wrapper for TarWithOptions.
```

Comment:
nit: may use markup while at it, i.e.
```suggestion
// stream of bytes. This is a convenience wrapper for [TarWithOptions].
```

so that go doc will link to this function.

Reply:
done.